interbotix_ros_xscobots/interbotix_xscobot_ros_control/config/controllers/dx400_controllers.yaml

```diff
@@ -0,0 +1,39 @@
+/$(var robot_name):
+  # This config file is used by ros2_control
+  controller_manager:
+    ros__parameters:
+      update_rate: 100  # Hz
+
+      arm_controller:
+        type: joint_trajectory_controller/JointTrajectoryController
+
+
+      gripper_controller:
+        type: joint_trajectory_controller/JointTrajectoryController
+
+
+      joint_state_broadcaster:
+        type: joint_state_broadcaster/JointStateBroadcaster
+
+  arm_controller:
+    ros__parameters:
+      joints:
+        - j1
+        - j2
+        - j3
+        - j4
+        - j5
+        - j6
+      command_interfaces:
+        - position
+      state_interfaces:
+        - position
+        - velocity
```

Comment:
Only position state interface is available in the XSHardwareInterface's current state